core/src/main/java/io/github/lsposed/lspd/service/ConfigManager.java

```diff
@@ -137,9 +148,17 @@ public int hashCode() {
 
     private final ConcurrentHashMap<Integer, String> cachedModule = new ConcurrentHashMap<>();
 
-    private void updateCaches() {
-        cacheScopes();
-        cacheModules();
+    private void updateCaches(boolean sync) {
+        synchronized (this) {
+            requestScopeCacheTime = requestModuleCacheTime = System.currentTimeMillis();
```

Comment:
Consider using SystemClock.elapsedRealtime because System.currentTimeMillis is never guaranteed to be monotonically
increasing.